django-rgd-imagery/rgd_imagery/templates/rgd_imagery/rastermeta_detail.html

```diff
@@ -53,6 +53,9 @@
   <select id="band-selector" onchange="updateBandThumbnail()">
   </select>
 
+  <select id="colormap-selector" onchange="updateBandThumbnail()">
+  </select>
```

Comment:
Can you make sure this is only visible when a band other than "default" is selected?